CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ### Features
 
 ### Fixes
-
+Added dbt-core 1.3.3 dependency to dbt-teradata 1.3.3 to remove the pytz dependency that was there in dbt-core 1.3.0
```

Comment:
Varun, can you please add original issue of pytz module not found issue link here

Reply:
Sure Satish